actors/eam/src/ext.rs

```diff
@@ -38,3 +38,7 @@ pub mod evm {
 
     pub const RESURRECT_METHOD: u64 = 2;
 }
+
+pub mod account {
+    pub const PUBKEY_ADDRESS_METHOD: u64 = 2;
```

Comment:
Even if this is a built-in actor and this authorised to call a reserved method, why not call the FRC42 endpoint?

Reply:
Why? This is internal code, why bother with the user space methods? We can use reserved methods just fine at this level.

Reply:
This is a "trusted" call (we need the target to actually be an account) so I'd keep using the private one for now.